### DISCUSSION TITLE
Legend size

### DISCUSSION THREAD

**Discussion by CalmRealistic** [ImmuneDynamics/Spectre#174] [! News]
Hi, 

How to increase the size of little color dots of the legend? 

I tried
make.multi.plot(CD3cell.sub, "tSNE_X", "tSNE_Y", CD3cellular.cols, dot.size=0.25)+
guides(color = guide_legend(override.aes = list(size = 5)))

It increases the symbol size on my screen, but not in the new saved image of the plot

Thank you

**Comment by csecarroll** [comment]
I've not had much luck with any customisation of the spectre plots, but would love to know if anyone else does. 

My guess to why it doesn't change the saved plot is because the plot would be saved when it runs the function in the first line, prior to adding the guides. 

I'd absolutely love some more customisation options on these plots.

**Comment by tomashhurst** [comment]
@CalmRealistic @csecarroll great question! I think in the next update we'll just add this as an argument in the function directly -- I too find the legend points too small.

One solution in the meantime: if you assign the plot first (and in this case switching off the auto save function) you should be able to use add classic `ggplot` arguments. You can then use `ggsave` to save the image once you are done

Here is an example:

```
## Setup
library('Spectre')
dat <- Spectre::demo.clustered
dat <- do.subsample(dat, 10000)

## Assign a plot to `p`
p <- make.colour.plot(dat = dat, x.axis = 'UMAP_X', y.axis = 'UMAP_Y', col.axis = 'Population', save.to.disk = FALSE) 
p

## Adjust the size of the dots in the legend but not the actual plot
p <- p + guides(color = guide_legend(override.aes = list(size = 10))) 
p

## Save the image
ggsave(filename = 'Test image.png',
       plot = p,
       path = getwd(),
       width = 9,
       height = 7,
       limitsize = FALSE)
```

Seems to work OK for me, give it a test run and see what happens.
